README.md

```diff
@@ -21,3 +21,7 @@ linking it, offering various features to control the build process. This crate
 is intended for developers who need direct access to the NNG C API or are
 building higher-level wrappers like `nng`. For more details, please see the
 [`nng-sys` crate's README](./nng-sys/README.md).
+
+## `anng`
+
+This crate provides safe, async Rust bindings for NNG.
```

Comment:
Think we should add a few words here when to use `nng` and `anng`. I'd also mention that it is tied to Tokio.